packages/axios-extension/src/abap/types/adt-types.ts

```diff
@@ -95,3 +95,9 @@ export type AdtTransportStatus = 'S' | 'E';
 // the <DLVUNIT/> element contain text that indicate it is local package.
 // No transport number required for deploying to local pacakge.
 export const LocalPackageText = ['LOCAL_PACKAGE', 'LOCAL'];
+
+export interface PackageInfo {
```

Comment:
JSDoc for this? Maybe include the purpose of this interface?

Reply:
Where is this interface used?

Reply:
Only used internally when processing response data from the ADT service. Updated code to avoid exposing this interface to axios-extension consumer.